### PR TITLE
Bump Node version to 8.13

### PR DESCRIPTION
**Pull request by whymarrh**
This PR bumps the Node version for nvm to 8.13 which now includes npm@6 (where 8.11.3 included npm@5). I've also updated the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,15 @@ If you're a web dapp developer, we've got two types of guides for you:
...
 
- - Install [Node.js](https://nodejs.org/en/) version 8.11.3 and npm version 6.1.0
-   - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
-   - Select npm 6.1.0: ```npm install -g npm@6.1.0```
- - Install dependencies: ```npm install```
- - Install gulp globally with `npm install -g gulp-cli`.
- - Build the project to the `./dist/` folder with `gulp build`.
- - Optionally, to rebuild on file changes, run `gulp dev`.
- - To package .zip files for distribution, run `gulp zip`, or run the full build & zip with `gulp dist`.
+- Install [Node.js](https://nodejs.org) version 8 and npm version 6
```
Comment (brunobar79) [inline README.md]: latest version 8? or maybe 8.13+ ?

Just trying to avoid users running 8.11.3 or lower that might mess this up

**Review by brunobar79 — DISMISSED** [review]
LGTM! just one optional comment

File: README.md (hunk 1 above)
Comment (whymarrh) [reply README.md]: The Node version isn't too important, the app should work with any 8.y.z. The npm version is the real kicker and if they update the npm version for whatever version they're using they should be a-ok.

Comment (brunobar79) [reply README.md]: That's fair. When I said "mess this up" I was referring to the package-lock.json getting massive changes if you're using different versions of npm 6. Maybe specifiy the latest npm version or 6.41.+ ?

Comment (whymarrh) [reply README.md]: Oh yes that's a good point. I'll update this.

**Review by brunobar79 — APPROVED** [review]
💯!

**Pull request merged**